docs/.vuepress/theme/components/ThemeSwitcher.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <label id="switch" class="switch">
+      <input type="checkbox" v-on:change="toggleTheme" id="slider" :checked="isDarkTheme">
+      <span class="slider round"></span>
+  </label>
+</template>
+
+<script>
+const CLASS_THEME_DARK = 'theme-dark';
+const SRC_LOGO_DEFAULT = '/img/handsontable-logo.svg';
+const SRC_LOGO_WHITE = '/img/handsontable-logo-white.svg';
+
+const getDomElement = (selector) => {
+  return document.querySelector(selector);
+};
+
+const toggleDarkThemeClassOnHTML = (htmlDomElement, isDarkTheme) => {
+  if (isDarkTheme) {
+    htmlDomElement.classList.add(CLASS_THEME_DARK);
+
+    return;
+  }
+
+  htmlDomElement.classList.remove(CLASS_THEME_DARK);
+};
+
+const changeLogo = (logoEl, isDarkTheme) => {
+  logoEl.src = isDarkTheme ? SRC_LOGO_WHITE : SRC_LOGO_DEFAULT;
+};
+
+export default {
+  name: 'ThemeSwitcher',
+  methods: {
+    toggleTheme() {
+      this.isDarkTheme = !this.isDarkTheme;
+      const theme = this.isDarkTheme ? 'dark' : 'light';
+
+      localStorage.setItem('theme', theme);
```

Comment:
Can I suggest feature detection here? If `localStorege` does not exist, do nothing. caniuse.com has some missing data but it's not that essential feature and we may make it a progressive enhancement for browsers that support it.